XO/XO.py

```diff
@@ -0,0 +1,116 @@
+from typing import List, Union, Tuple
+import random
+
+N = 3
+EMPTY = 'E'
+PLAYERS_SIGNS = ['X', 'O']
+
+
+def get_names() -> List[str]:
+    basic_text = 'Insert the name of player {index}\n'
+    inputs = []
+    for i in range(2):
+        text = basic_text.format(index=i)
```

Comment:
An even shorter (and nicer in my opinion) way, will be:
`inputs.append(input(f'Insert the name of player {i}\n'))`
(1 line instead of 3)